group/views.py

```diff
@@ -22,25 +22,23 @@
 SOFTWARE.'''
 from django.http import HttpResponse
 from django.shortcuts import render_to_response, render
+from group.models import Group
+import datetime
+
+def get_running_contest(request, group_id):
+        
+        group = Group.objects.get(id = group_id)
+        all_contest = group.trace_contest.all()
+        all_running_contest_list = []
+        now = datetime.datetime.now()
+
+        for contest in all_contest:
+            start_time = contest.end_time.replace(tzinfo=None)
```

Comment:
+ from django.utils import timezone
+ now = timezone.now()

##### This will let you compare(line 38,39) successfully without modifying timezone like (replace tzinfo=None)